### ISSUE TITLE
Paper inconsistencies

### ISSUE THREAD

**Issue by bobetocalo** [wywu/LAB#20]
Dear Wayne Wu,

First of all I would like to congratulate you for your excellent work. I'm a PhD student at Spain. My research is focused on face alignment. I have used your WFLW trained model successfully and I read your CVPR paper. I would like to ask some questions:

- Training. I am not sure which training images are you using in your experiments in COFW-29 and AFLW? I understand that in the LAB result you have trained using the 300W images to supply boundary information ... consequently it is not comparable with the literature. On the other hand, in the LAB w/o boundary result, how is it possible to train such a complex DCNN (res-18 architecture) using only 1345 training images in COFW? Are you using fine-tuning or training from scratch?

- Testing. Your results in the 300W table are inconsistent between pupils and corners normalization. For example, according to literature it is not possible to obtain in the challenging subset a NME of 6.98 (pupils) and  5.19 (corners).

|Method|Pupils|Corners|
|:--:|:--:|:--:|
|LAB|**6.98**|**5.19**|
|SHN [paper](http://openaccess.thecvf.com/content_cvpr_2017_workshops/w33/papers/Yang_Stacked_Hourglass_Network_CVPR_2017_paper.pdf)|**7.00**|4.9|
|DAN [paper](https://arxiv.org/pdf/1706.01789.pdf)|7.57|**5.24**|

Do you have an explanation for this normalization error? Could you update your fixed results or release the trained model in 300W?

I look forward to your response.

Best regards,
Roberto Valle

**Comment by ghost** [comment]
I feel the same thing, can the author explain it?

**Comment by wywu** [comment]
Hi, 

Thanks for your interest!
1. we report the results in Table 4 in order to evaluate the ability of cross-dataset auxiliary training of our method rather than comparison.  LAB w/o boundary is obtained by a Res-18 modified by adding two
fully-connected layers with 256 units and cut to left only a quarter of feature maps on each layer. We trained from scratch and dropout with 0.5 is used before the first fully-connection layer.
2. We are sorry that some details are missed in our original version of the paper. The inconsistency is caused by the different stacks of hourglass are used in the model. We have updated the results in Table. 1 in this version: https://wywu.github.io/projects/LAB/support/LAB.pdf. We will upload it to Arxiv later.

Best,
Wayne
